.gitignore

```diff
@@ -3,4 +3,7 @@
 
 # Ignore Personal GIMP Files
 .var/app/org.gimp.GIMP/data
-.var/app/org.gimp.GIMP/current
\ No newline at end of file
+.var/app/org.gimp.GIMP/current
```

Comment:
These (data, current) folders are generated by Gimp on startup and every project created, that contains only files related to that user, so isn't necessary.